modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java

```diff
@@ -266,6 +284,38 @@ private ObjectField _toObjectField(
 			objectField);
 	}
 
+	private void _validateAndCreateListTypeDefinition(ObjectField objectField)
+		throws Exception {
+
+		if (Objects.equals(
+				objectField.getBusinessTypeAsString(),
+				ObjectFieldConstants.BUSINESS_TYPE_PICKLIST)) {
+
+			if (Objects.isNull(
```

Comment:
@murilostodolni you can use the `Validator.isNull`/`Validator.isNotNull` instead